staging/src/k8s.io/client-go/tools/clientcmd/loader_test.go

```diff
@@ -623,7 +651,10 @@ func TestMigratingFileSourceMissingSkip(t *testing.T) {
 	sourceFilename := "some-missing-file"
 	destinationFile, _ := ioutil.TempFile("", "")
 	// delete the file so that we'll write to it
-	os.Remove(destinationFile.Name())
+	destinationFile.Close()
+	if err := os.Remove(destinationFile.Name()); err != nil {
+		t.Fatalf("failed to remove %v.err:%v", destinationFile.Name(), err)
```

Comment:
```suggestion
		t.Fatalf("failed to remove file %v: %v", destinationFile.Name(), err)
```

at minimum, add a space after `:`

Reply:
same in other places

Reply:
Thank you for your reminder, it's done.